### PR TITLE
Option to hide minimap

### PR DESCRIPTION
**Pull request by Blizik**
Adds toggle to minimap plugin to hide the minimap entirely. Covers most of #614. With the option to hide orbs in game, adding an option to hide orbs individually seemed like too much clutter, and a little redundant.

### CODE REVIEW

File: runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java [hunk 1]
```diff
@@ -178,6 +178,7 @@
 
 	static class Minimap
 	{
+		static final int MINIMAP_WIDGET = 17;
```
Comment (Adam-) [inline runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java]: This class is supposed to be for widget group 160, and this is not a part of that

**Comment by Adam-** [comment]
This doesn't work at all on fixed mode. I think the widget is different.

**Comment by Blizik** [comment]
My understanding of the feature request was that they wanted more screen space when the minimap wasn't necessary. I didn't test it, but I assumed hiding the minimap wouldn't open more screen space on fixed more.

**Comment by Adam-** [comment]
Ah, that makes sense. Okay. Maybe change the config item description to state it is only for resizable.

File: runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java (hunk 1 above)
Comment (Blizik) [reply runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java]: Should I use the Viewport class instead?

**Pull request merged**